schemas/dab.draft.schema.json

```diff
@@ -207,6 +233,16 @@
           "description": "Global hosting configuration",
           "additionalProperties": false,
           "properties": {
+            "max-response-size-mb": {
+             "type": [
+                "integer",
+                "null"
+              ]
+              "description": "Specifies the maximum size, in megabytes, of the database response allowed in a single result. If set to null, the default value is 158 MB.",
```

Comment:
Why is default value = 158 MB? Why not some power of 2 for e.g. 128 or 256?

Reply:
See #2242:
https://github.com/Azure/data-api-builder/blob/f97aa96b70371ea00ea9d320b6deecde1ebde0c5/src/Config/ObjectModel/HostOptions.cs#L11-L21